features/admin/invite_admin_user.feature

```diff
@@ -0,0 +1,25 @@
+@invite-admin-user
+Feature: Admin manager can manage users
+
+Scenario: Admin Manager user can log in and invite admin users
+  Given I am logged in as the production admin-manager user
+  And I am on the /admin page
+  And I click the 'Manage users' link
+  And I click the 'Invite user' link
+  When I enter 'functional-test@user.marketplace.team' in the 'Email address' field
+  #TODO change this to 'Manage services' once radio buttons are updated
+  And I choose the 'Category' radio button
+  And I click the 'Invite user' button
+  Then I take a screenshot
```

Comment:
No need for the screenshotting here.  Is there a success banner to check for on the page instead?

Reply:
Done 👍